source-code/core/plugin/src/api.test-d.ts

```diff
@@ -25,26 +25,24 @@ const plugin1: Plugin<PluginOptions, AppSpecificApis> = {
 		description: { en: "i18next plugin for inlang" },
 		keywords: ["i18next", "react", "nextjs"],
 	},
-	setup: ({ options }) => {
-		if (options.pathPattern === undefined) {
-			throw Error("Path pattern is undefined")
-		}
-		return {}
-	},
 
 	loadMessages: async () => {
 		return []
 	},
 	saveMessages: async ({ messages }) => {
-		console.log(messages)
+		return
 	},
 	addAppSpecificApi: () => {
 		return {
 			"inlang.ide-extension": {
-				messageReferenceMatcher: () => {},
+				messageReferenceMatcher: () => {
+					return
+				},
 			},
 			"inlang.cli": {
-				cliExtract: () => {},
+				cliExtract: () => {
+					return
+				},
```

Comment:
the cli needs extract?

Reply:
no whatever this is just a random test 😄